src/utils/worker/createResponseListener.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  ServiceWorkerIncomingEventsMap,
+  SetupWorkerInternalContext,
+} from '../../setupWorker/glossary'
+import { ServiceWorkerMessage } from '../createBroadcastChannel'
+
+export function createResponseListener(context: SetupWorkerInternalContext) {
+  return (
+    _: MessageEvent,
```

Comment:
Is there a reason you named this parameter ` _`?

Reply:
To satisfy the linting rule, denoting that the first argument here (`event`) is not used.